File: applications/spring-geode-showcase/src/main/java/com/vmware/spring/geode/showcase/repostories/AccountRepository.java
package com.vmware.spring.geode.showcase.repostories;

import com.vmware.spring.geode.showcase.domain.Account;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AccountRepository extends CrudRepository<Account,String>
{
}

File: components/bank-account-service/src/main/kotlin/com/vmware/financial/open/banking/account/repository/AccountRepository.kt
package com.vmware.financial.open.banking.account.repository

import com.vmware.financial.open.banking.domain.account.BankAccount
import org.springframework.data.keyvalue.repository.KeyValueRepository
import org.springframework.data.repository.CrudRepository


    interface AccountRepository : KeyValueRepository<BankAccount, String> {
    }

